assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -400,21 +447,23 @@ subroutine validate_inflate_options(inf_flavor, inf_damping, inf_initial_from_re
 end do
 
 ! Check to see if state space inflation is turned on
-if (inf_flavor(1) > 1) do_prior_inflate     = .true.
-if (inf_flavor(2) > 1) do_posterior_inflate = .true.
+if (inf_flavor(PRIOR)     > OBS_INFLATION)  do_prior_inflate     = .true.
+if (inf_flavor(POSTERIOR) > OBS_INFLATION)  do_posterior_inflate = .true.
```

Comment:
this code isn't safe - it assumes OBS_INFLATION has value 1 and would break if changed.  to be more clear, maybe define a separate parameter, value = 2 , which is named 'STATE_SPACE_INFLATION_START' or something and make this code >=